src/views/Cart/ProductQuantity/ProductQuantity.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from "react";
+import "./ProductQuantity.scss";
+
+const ProductQuantity = () => {
+    const [quantity, setQuantity] = useState(0);
+
+    const plusQuantity = () => {
+      const newQuantity = quantity + 1;
+      setQuantity(newQuantity);
+    };
+  
+    const minusQuantity = () => {
+      if (quantity === 0) return;
+  
+      const newQuantity = quantity - 1;
+      setQuantity(newQuantity);
+    };
+
+    return(
+        <div className="product-cart-quantity">
+        <button
+          type="button"
+          onClick={minusQuantity}
+          className="product-cart-quantity--button"
```

Comment:
```suggestion
          className="product-cart-quantity__button"
```
button do element w bloku product-cart-quantity a nie jego modyfikator

https://www.nafrontendzie.pl/metodyki-css-2-bem